src/encoded/static/components/viz/AboutPageVisualizations/Alluvial/sankey.js

```diff
@@ -353,8 +353,18 @@ export function sankeyFunc() {
             return a.category.localeCompare(b.category);
         }
 
-        function ascendingDepth(a, b) {
-            return a.y - b.y;
+        function descendingDepth(a, b) {
+            return b.dy - a.dy;
+        }
+
+        function descendingDepthCategorical(a, b) {
+            if (b.dy - a.dy === 0 && b.type === 'assay_type') {
+            }
+            let a_group = a.assay_group.split('-');
+            let b_group = b.assay_group.split('-');
+
+            return a_group[0] - b_group[0] || a_group[1] - b_group[1];
+            // return b.dy - a.dy;
         }
```

Comment:
Maybe remove this comment?

Reply:
Thank you for catching this, will remove